Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogHeaderWidget.cpp

```diff
@@ -523,6 +523,7 @@ namespace O3DE::ProjectManager
         gemMenu->addAction( tr("Show Gem Repos"), [this]() { emit OpenGemsRepo(); });
         gemMenu->addSeparator();
         gemMenu->addAction( tr("Add Existing Gem"), [this]() { emit AddGem(); });
+        gemMenu->addAction( tr("Create a new Gem.."), [this]() { emit CreateGem(); });
```

Comment:
nit: the `..` seems like something we don't typically do in our menus, also we should keep the capitalization consistent
```suggestion
        gemMenu->addAction( tr("Create New Gem"), [this]() { emit CreateGem(); });
```